README.md

```diff
@@ -1,21 +1,32 @@
-
 # Playlister CLI
 
-In [playlister-rb](http://learn.flatironschool.com/lessons/940), you built a simple domain model for Artists, Songs, and Genres and in [OO Jukebox](http://learn.flatironschool.com/lessons/806), you built an Object Oriented CLI Jukebox. It is now time to combine these to build an interactive Command Line Jukebox that is populated with data from a folder filled with MP3s.
+## Objectives
 
-## Setting Up the Project
+1. Build classes that are associated to one another via has many, belongs to and through associations. 
```

Comment:
so has_many/belong_to are not being introduced yet as active record methods. this is just basic object orientation, no database. just introducing the concept so that when active record comes along, no one's head explodes. do you think i should still use back ticks?